ui/webui/src/components/AnacondaHeader.jsx

```diff
@@ -64,6 +66,7 @@ export const AnacondaHeader = ({ beta, title, reportLinkURL }) => {
     return (
         <PageSection variant={PageSectionVariants.light}>
             <Flex spaceItems={{ default: "spaceItemsSm" }} alignItems={{ default: "alignItemsCenter" }}>
+                {!isBootIso && <span className="logo" />}
```

Comment:
I think we should just make sure the icon is available on both boot.iso and Live image, even if we have to bundle it for now.

Reply:
https://github.com/weldr/lorax/pull/1341 is now merged so in a few days there will no be any broken symlink. 
So your concern will be solved.